.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+#### Expected behavior vs actual behavior
+<br><br>
+
+#### Steps to reproduce
+*(e.g., detailed walkthrough, runnable script, example application)*
+<br><br><br>
+
+#### Environment
+
+AMS Version *(commit ref if not on tag)*: 
+
+Ruby Version & Patch Level: 
+
+OS Type & Version: 
```

Comment:
Alternatively, we can do something like I did in metric_fu https://github.com/metricfu/metric_fu/blob/6487e77c340074a2c9de4e535c8a54ee9ae2a3b6/lib/metric_fu/environment.rb#L112

there's no command-line thing from ruby that I particularly like though, except maybe `ruby -e "puts RUBY_PLATFORM"`